test/release/examples/primers/chplvis/prog2.chpl

```diff
@@ -0,0 +1,44 @@
+// Example 2 of use of VisualDebug module and chplvis tool.
```

Comment:
I think a user encountering the chplvis directory in the primers directory will first look at the .chpl files, given that is where the entirety of the other information lived.  Placing this tutorial in a different place would give you more freedom to direct the order in which the files will be read.  I think it does make sense to have the majority of the information in the .html file, especially for the call outs to the pictures provided, and I think for that reason it would be better to put this tutorial in a different place, so that its flow is properly respected.